test/ipc-ut.cc

```diff
@@ -25,13 +25,11 @@ TEST(IPCTest, Positive) {
   // Fork
   pid_t child_pid = fork();
   if (!child_pid) {
-    std::string fileName = UNIT_TEST_DATA "/ipc_test_data_Positive.txt";
+    FILE *tmp_file = std::tmpfile();
```

Comment:
I think this is good.  Arguably, we should do a better job cleaning up the temporary files in the face of a catastrophic failure.  The standard doesn't specify what needs to happen to the temporaries if the process is closed abruptly when the file is open.  In theory we can `unlink()` the path immediately after calling `fileno()`, but I don't know how much value there is in doing that.